polaris-react/src/components/Link/Link.tsx

```diff
@@ -14,8 +13,6 @@ export interface LinkProps {
   url?: string;
   /** The content to display inside the link */
   children?: React.ReactNode;
-  /** Makes the link open in a new tab */
-  external?: boolean;
```

Comment:
This prop will need to be deprecated first and then we can remove it in v12.

```suggestion
  /**
    ** @deprecated Set `target` to "_blank" instead
    ** Makes the link open in a new tab
  */
  external?: boolean;
```